controlpanel/frontend/forms.py

```diff
@@ -1,17 +1,22 @@
+import re
+
 from django import forms
 from django.conf import settings
 from django.contrib.postgres.forms import SimpleArrayField
 from django.core.exceptions import ValidationError
-from django.core.validators import RegexValidator
+from django.core.validators import RegexValidator, validate_email
 
 from controlpanel.api import validators
 from controlpanel.api.cluster import get_repository
-from controlpanel.api.models import S3Bucket
+from controlpanel.api.models import App, S3Bucket
 from controlpanel.api.models.access_to_s3bucket import S3BUCKET_PATH_REGEX
 from controlpanel.api.models.iam_managed_policy import POLICY_NAME_REGEX
 from controlpanel.api.models.parameter import APP_TYPE_CHOICES
 
 
+APP_CUSTOMERS_DELIMITERS = re.compile(r'[,; ]+')
```

Comment:
So this is to strip `[`, `,`, `;`, ` `, and `]` chars right? Have you seen anyone pasting angle brackets (`<`,`>`)? Maybe Auth0 is angle-brackets-friendly 🤷‍♂ ?

Reply:
I haven't seen it, but I agree that is quite a likely mistake to make. Those angle backets would not be delimiters, though, so a different regex would be needed to sanitize the inputs. I'm not sure how far we should go to be lenient in what we accept from user input. I think in this case, you are right and we can sanitize emails in the form `Firstname Lastname <username@domain>`.

Reply:
Agreed, probably out of scope and if it's a problem we could look at that separately (or maybe even add something in the client code to explain the input is invalid/prevent the submit ... in the future)